sphinx/source/docs/user_guide/embed.rst

```diff
@@ -227,10 +227,10 @@ For example, to use version ``1.4.0``, including widgets and tables support:
 .. note::
     You must provide the closing `</script>` tag. This is required by all
     browsers and the page will typically not render without it. You should also
-    always include the `crossorigin="anonymous"` attribute on the script.
+    always include the `crossorigin="anonymous"` attribute in the script.
```

Comment:
it is actually an attribute *on*, but on the script tag

Reply:
Very good point, I'll change it to: "You should also always include the `crossorigin="anonymous"` attribute on the script tag."